style.css

```diff
@@ -257,6 +257,26 @@ header {
     padding: 0;
 }
 
+.delete-button { 
```

Comment:
hi Sam, 
I see only line 261, line 276 and line 277 are effective. Please remove these lines.
Others are either overwritten by Bootstrap classes in the script.js.
Line269 to 274 are not relevant at all.